src/app/users/users.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, inject } from '@angular/core';
+import { User } from 'src/Model/user.interface';
+import { UserService } from '../Services/User.service';
+
+@Component({
+  selector: 'app-users',
+  templateUrl: './users.component.html',
+  styleUrls: ['./users.component.css']
+})
+export class UsersComponent implements OnInit {
+  
+  user!: User;
```

Comment:
You have to return a list of user not a single user, because the json file contains a list of users, not a single user

Reply:
thanlk you its all ok i did all your comments and all good now should merge or pull anthor request